### PR TITLE
[eslint] prefer object dot notation

### PR DESCRIPTION
**Pull request by dimaMachina**

### CODE REVIEW

File: packages/web/app/src/env/read.ts [hunk 1]
```diff
@@ -1,13 +1,10 @@
-function isBrowser() {
-  // eslint-disable-next-line no-restricted-syntax
-  return Boolean(
-    typeof window !== 'undefined' && '__ENV' in window && window['__ENV'] !== undefined,
-  );
+function isBrowser(): boolean {
+  return typeof window !== 'undefined' && '__ENV' in window && window.__ENV !== undefined;
```
Comment (dimaMachina) [inline packages/web/app/src/env/read.ts:2]: unneeded cast to `boolean`

**Comment by kamilkisiela** [comment]
I will merge it if you give me two arguments how it affects Hive users :)

**Comment by dimaMachina** [comment]
- is easier to read
- less verbose
- works better with aggressive JavaScript minimizers

**Comment by kamilkisiela** [comment]
> is easier to read
> less verbose

just a personal preference

> works better with aggressive JavaScript minimizers
we don't really care about 20b improvement in bundle size

not enough to add new eslint rules, I'm sorry

**Pull request closed without merging**